src/dune_rules/obj_dir.ml

```diff
@@ -406,7 +400,7 @@ module Module = struct
     let basename =
       Module_name.Unique.artifact_filename obj_name ~ext:".odocl"
     in
-    relative t (odocl_dir t) basename
+    relative t (odoc_dir t) basename
```

Comment:
I know function `odocl` is supposed to be used some where, but I’m not sure of where exactly.